package.json

```diff
@@ -21,6 +21,6 @@
   "devDependencies": {
     "eslint": "^4.15.0",
     "foreman": "https://github.com/pryv/node-foreman/archive/master.tar.gz",
-    "simon-say": "https://github.com/andrewn/simon-say/archive/master.tar.gz"
+    "simon-say": "https://github.com/andrewn/simon-say/archive/eb6d4cc.tar.gz"
```

Comment:
You might be better off referring to a tag or a branch here, so that you can push an update without having to change this dependecy

Reply:
Does that work? It didn't detect new commits on `master`, which is why I changed to a specific commit.

Reply:
I meant that NPM didn't detect new commits on master, when I did `npm install` after `simon-say` had been installed once.